app.js

```diff
@@ -22,17 +25,26 @@ app.all("*", (req, res) => {
 
 app.use((err, req, res, next) => {
   if (err.code === "22P02") {
-    res.status(400).send({ msg: "review_id must be a number" });
+    let value = "";
+    if (req.method === "GET") value = " for review_id";
```

Comment:
this logic might not work when you add the tests for invalid id for a patch request